hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestDataSourceForBootstrap.scala

```diff
@@ -475,8 +478,8 @@ class TestDataSourceForBootstrap {
   }
 
   def runMetadataBootstrapAndVerifyCommit(tableType: String,
-                                          partitionColumns: Option[String] = None,
-                                          extraOpts: Map[String, String] = Map.empty): String = {
+      partitionColumns: Option[String] = None,
```

Comment:
no need to change the format?

Reply:
this's a personal practice that format codes involved in this commit. i recover this.